packages/uploadthing/src/internal/handler.ts

```diff
@@ -223,7 +223,7 @@ export const buildRequestHandler = <
       return { status: 200 };
     }
 
-    if (!actionType || actionType !== "upload") {
+    if (!actionType || !["upload", "failure"].includes(actionType)) {
```

Comment:
Could we perhaps move this array out, mark it as const and then infer the type from it and use that in `createRequestUrl`

Reply:
Also the error below is outdated